src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -102,12 +123,14 @@ Search Console is well-suited for both developers and those in non-developer rol
 
 Search Console may not be a fit for projects that use different third-party tools which group pages by similarity, or if a website isn't represented in the CrUX dataset.
 
+Page grouping can also be somewhat confusing when the example pages in that group have different characteristics of the group. For example, if the group fails particular Core Web Vitals overall, but the example pages all seem to pass the same Core Web Vitals. This can happen when a group contains a long tail or little visited pages that may be slower to load as less likely to be cached. When there are sufficient volumes of these pages in the long tail, they can influence the group's overall pass rate.
```

Comment:
```suggestion
Page grouping can also be somewhat confusing when the example pages in a group have different characteristics than the rest of the group—for example, if the group fails particular Core Web Vitals overall, but the example pages all seem to pass the same Core Web Vitals. This can happen when a group contains a long tail or seldomly visited pages that may be slower to load, as they're less likely to be cached. When there are sufficient volumes of these pages in the long tail, they can influence the group's overall pass rate.
```